docs/uplink/create-tunnels.md

```diff
@@ -230,6 +230,49 @@ Content-Length: 973
 </pre>
 ```
 
+#### Multiple upstreams
+
+What if the customer is running multiple HTTP services that you need to access? It is possible to expose multiple upstream services over a single tunnel.
```

Comment:
> What if the customer is running multiple HTTP services that you need to access? 

The following example shows how to access more than one HTTP service over the same tunnel.